build.gradle

```diff
@@ -45,10 +45,10 @@ checkstyle {
     ignoreFailures = false // Whether this task will ignore failures and continue running the build.
     configFile rootProject.file('config/checkstyle/checkstyle.xml')
     // The Checkstyle configuration file to use.
-    toolVersion = '8.29' // The version of Checkstyle you want to be used
+    toolVersion = '9.1' // The version of Checkstyle you want to be used
 }
 
-def hasPyCharm = project.hasProperty('pycharmPath')
+def hasPycharmPath = project.hasProperty('pycharmPath')
 def hasPythonPlugin = project.hasProperty('pythonPlugin')
```

Comment:
Is this still needed if, as you said, the plugin is now mandatory no matter the IDE used?

Reply:
Good point, I don't think it's useful. Will remove.